### DISCUSSION TITLE
You have unread notifications. But I don't have.

### DISCUSSION THREAD

**Discussion by antgubarev** [community/community#52019] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The problem is on the screenshots bellow.
<img width="915" alt="Screenshot 2023-04-05 at 08 47 04" src="https://user-images.githubusercontent.com/2937969/229992044-7f34fc37-68da-4141-8773-4fd2d27c2983.png">
<img width="298" alt="Screenshot 2023-04-05 at 08 46 46" src="https://user-images.githubusercontent.com/2937969/229992053-3f9771af-4115-483a-8fab-dc99bd550845.png">

**Comment by csbun** [comment]
I have the same issue. The only difference is the "unread notification" comes from a different repository.

**Comment by radar-blip** [comment]
Here is what works for a lot of people:

https://github.com/orgs/community/discussions/6874#discussioncomment-3593547

**Reply by csbun** [reply]
https://github.com/orgs/community/discussions/6874#discussioncomment-4603794 works for me
